README.md

```diff
@@ -4,25 +4,33 @@ Welcome to the new Word Javascript API! We hope you enjoy it an find it useful.
 
 ## Release Notes 
 
-This is the first release of JavaScript APIs for Word and we focused on the following functional areas:
- 1. **Basic document navigation:** On top of having access to the user's current selection, we are also providing ways to traverse the document by exposing collection of two of the most important objects in Word: Paragraphs and Content Controls, having easy access to the content of the entire document.
+For this release of the JavaScript APIs for Word we focused on the following functional areas:
+ 1. **Basic document navigation:** On top of having access to the user's current selection, we are also providing ways to traverse the entire document by exposing collections of two of the most important objects in Word: Paragraphs and Content Controls.
```

Comment:
**Basic document navigation:** You can access the collection of Paragraph and Content Controls. 

Is the user's current selection something that is already well-understood. If not, just add on to the list after Content Controls.